src/sql/workbench/contrib/notebook/browser/cellViews/cellToolbar.component.ts

```diff
@@ -96,7 +99,14 @@ export class CellToolbarComponent {
 
 		let taskbarContent: ITaskbarContent[] = [];
 		if (this.cellModel?.cellType === CellTypes.Markdown) {
-			taskbarContent.push({ action: this._editCellAction });
+			taskbarContent.push(
+				{ action: splitCellButton },
```

Comment:
As discussed in the review - please move the split cell to not be the first item in the list

Reply:
@rajeshka Either one of us can move this. Let me know if you need help.

Reply:
@halerankin  - You can take it up as my knowledge here is limited.

Reply:
Done